src/bin/utils/mod.rs

```diff
@@ -470,4 +491,70 @@ mod tests {
     fn test_jitvalue_to_felt_null() {
         assert_eq!(jitvalue_to_felt(&JitValue::Null), vec![Felt::ZERO]);
     }
+
+    #[test]
+    fn test_jitvalue_to_felt_felt252_dict() {
+        let result = jitvalue_to_felt(&JitValue::Felt252Dict {
+            value: HashMap::from([
+                (Felt::ONE, JitValue::Felt252(Felt::from(101))),
+                (
+                    Felt::TWO,
+                    JitValue::Array(Vec::from([
+                        JitValue::Felt252(Felt::from(201)),
+                        JitValue::Felt252(Felt::from(202)),
+                    ])),
+                ),
+            ]),
+            debug_name: None,
+        });
```

Comment:
Why does the dict in this test have two possible types for its value (array & felt), this is not supported in cairo

Reply:
Yes, I know, it was just a unit test example to check the two types of key,value types.
I divided it into two different unit tests